package.json

```diff
@@ -206,6 +215,14 @@
       }
     ],
     "views": {
+      "malloy": [
+        {
+          "id": "malloyHelp",
+          "name": "Help",
+          "type": "webview",
+          "when": "malloy.webviewPanelFocused == true || resourceFilename =~ /\\.malloy$/"
```

Comment:
I don't think we want this restriction, we want the icon to show up when Malloy is installed, not just when a Malloy file is open.

Reply:
Changed the Malloy is installed condition to `extensionStatus == installed`.